averages.js

```diff
@@ -0,0 +1,45 @@
+function total(input){
+  sum = 0
```

Comment:
remember when declaring variables you should use `var` or `let`/`const` to avoid creating unwanted globals